ROADMAP.md

```diff
@@ -77,30 +75,61 @@ Any dates listed below and the specific issues that will ship in a given milesto
     * Clouds that don't have code gen pipelines
       * Wrap stateless Terraform providers [#262](https://github.com/crossplane/crossplane/issues/262)
+      * VMWare vSphere Provider (v1alpha1)
+        * using codegen & stateless Terraform Providers
+        * https://github.com/crossplane-contrib/provider-terraform-vsphere
 
-  * GCP Provider
-    * Explore code generation of a native Crossplane provider-gcp
     * GCP: DNS, SSL, and Ingress support #1123 [#1123](https://github.com/crossplane/crossplane/issues/1123)
     * GCP storage buckets to v1beta1 [crossplane/provider-gcp#130](https://github.com/crossplane/provider-gcp/issues/130)
 
-  * Expanded Rook support
-    * Support additional Rook storage providers
-    * Install & configure Rook into a target cluster
-
   * Additional providers being incubated in https://github.com/crossplane-contrib
 
-* GitLab Auto DevOps Phase 2 - provision managed services from GitLab pipelines
+* GitLab Integration with Crossplane v1.0+
   * Currently the auto deploy app only supports PostgreSQL DBs
   * Support additional managed services from GitLab ADO pipelines
   * Add support for MySQL, Redis, Buckets, and more.
+  * Upgrade to Crossplane v1.0+ with XRDs & Compositions with default cloud service catalogs
 
 * Ease-of-use and improved experience
   * Standalone mode allowing Crossplane to run in a single container or process [#274](https://github.com/crossplane/crossplane/issues/274)
 
 ## Released
 
+### v1.1.0 Composition enhancements, provider code gen, security, and rate limiting
+
+* General
+  * Support Fs and Env ProviderConfig credential sources [crossplane-runtime#236](https://github.com/crossplane/crossplane-runtime/issues/236), [#2070](https://github.com/crossplane/crossplane/issues/2070) 
+    * AWS [provider-aws#518](https://github.com/crossplane/provider-aws/pull/518)
+    * Azure [provider-azure#218](https://github.com/crossplane/provider-azure/pull/218)
+    * GCP [provider-gcp#301](https://github.com/crossplane/provider-gcp/pull/301)
+  * [Guide for using Vault for Provider credentials](https://crossplane.io/docs/master/guides/vault-injection.html)
+  * Rate limiting support in the `ManagedReconciler` [#40](https://github.com/crossplane/crossplane-runtime/issues/40) - see related [blog post](https://danielmangum.com/posts/controller-runtime-client-go-rate-limiting/)
+
+* Composition
+  * Bi-directional patching for status [#1639](https://github.com/crossplane/crossplane/issues/1639)
+  * Support taking values from members to fill a connection secret [#1609](https://github.com/crossplane/crossplane/issues/1609)
+  * Support for required `fromFieldPath` patches [#2094](https://github.com/crossplane/crossplane/issues/2094)
+  * Support for reordering resources in a Composition template [#1909](https://github.com/crossplane/crossplane/issues/1909)
+  * Experimental `CustomComposition` [#1995](https://github.com/crossplane/crossplane/issues/1995)
```

Comment:
is there more information on how folks can play/experiment with this custom composition?

Reply:
yes, will add: https://github.com/eladb/cdk8s-operator/pull/16